### ISSUE TITLE
Allow checking for updates without syncing the database

### ISSUE THREAD

**Issue by donbex** [ouuan/pacwatch#12]
Currently, `pacwatch` checks for updates by calling `pacman` (or a `pacman` wrapper) with options `--noconfirm --color=always -Sy`. This means that the package database will always be synced, which can leave the system in an inconsistent state if the user chooses _not_ to update.

Therefore, it would be desirable to have the option to check for updates without syncing the package database, which could be achieved for example by using the `checkupdates` script from `pacman-contrib`.

Note that simply setting the value of the `pacman_command` option to `checkupdates` does not work, because that script does not support any of the options that `pacwatch` passes to `pacman`.

**Comment by ouuan** [comment]
How harmful an "inconsistent state" could be :thinking:? I can only think of running `pacman -S` in an "inconsistent state" and resulting in a partial upgrade. My suggestion is to always run a `pacman -Syu` before any `pacman -S`. So I'd say the "inconsistent state" is harmful but only a little bit.

Another consideration is that the result of `checkupdates` could be different from the later `pacman -Sy`. Although this inconsistency has a low probability, I do think it's more important than the harm of a `pacman -Sy` without `u`.

**Comment by ouuan** [comment]
UPD: I just realized that we can compare the result of `checkupdates` and `pacman -Sy`.

**Comment by ouuan** [comment]
OK, maybe using `checkupdates` is a good choice, but I don't think there's enough motivation for it. A PR would be welcome :)

**Comment by donbex** [comment]
Fair enough. Unfortunately right now I am unable to submit a PR due to my employment contract, but if/when I clear it with my employer I am happy to open one.

Either way, thanks for the quick response!
